### PR TITLE
minimal version

### PR DESCRIPTION
**Pull request by uniqueg**
I have removed the hashsplitter workflow, the WESkit part from the demo notebook, and some of the requirements.

Moreover, I have included the workflow and request/data flow schemas in the notebook, updated the text a bit and also updated the `README.md` (e.g., the S3 entry in the table; maybe we need to update the used service versions, like proTES?).

@vschnei: Maybe you could try this out? And if so, maybe you could merge it into your branch so that your PR gets updated? We could then have a final check and merge it into `main`. I don't think we need to worry about losing anything from the previous state - we can always go back to an older commit. The only exception: The stuff you added to the notebook about WESkit. Maybe you need to save that somewhere else (maybe best to immediately create a PR for a new demo in the other repo).

### CODE REVIEW

File: demo.ipynb
Comment (vschnei) [inline demo.ipynb:1]: For both requests we need to pass all the envs: 
envvars HOME ACCESS_KEY_ID SECRET_ACCESS_KEY ENDPOINT_URL BUCKET_PATH

**Comment by vschnei** [comment]
The rest looks fine to me
Sorry for the late review. I had it actually already on friday.

File: demo.ipynb
Comment (uniqueg) [reply demo.ipynb:1]: Where to put them? In .env? Maybe you could add it, maybe easier. Guess you could add it either here by pushing to my branch - or merge it first into your branch and then change it there.

**Comment by vschnei** [comment]
For visibility reasons, I have added them to the demo.ipynb.

**Pull request merged**